client/src/locales/translations/hi_IN.json

```diff
@@ -2,7 +2,7 @@
   "headline": "मदद देने और अनुरोध करने का स्थान",
   "tagline": "<0>महामारियां होती रहेंगी।</0><1> हम समुदायों को उनके लिए तैयार रहने और उनसे लड़ने में  मदद करते हैं।</1>",
   "common": {
-    "getHelp": "सहायता का अनुरोध करें",
+    "getHelp": "सहायता के लिए अनुरोध करें",
```

Comment:
@ddveloper Can you guide me on how to ensure that Localise is updated and synced? I am not sure how to do that.

Reply:
@Jameel139 , please advice 😃

Reply:
@ddveloper @hanzala-sohrab Yes, any changes to languages that are _not_ English US has to be done directly on Lokalise. Only English US can be changed directly in the .json file (we have it set up so that whenever there is a change made to the en_us.json file, it sends those changes to Lokalise). 
@hanzala-sohrab, please join the #translations channel on Slack and reach out to @Jameel139! He can set you up with a Lokalise account :)

Reply:
@ddveloper @doheekim93 I have made the necessary changes on Lokalise as well.